apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx

```diff
@@ -181,9 +197,17 @@ export const ModerationNotification = (props) => {
       : translated.commentUnhighlighted
 
     if (error) {
-      props.onChangeStatus(error)
+      setAlert({
+        type: 'error',
+        message: error,
+        timeInMs: 3000
+      })
     } else {
-      props.onChangeStatus(alertMessage)
+      setAlert({
+        type: 'success',
+        message: alertMessage,
+        timeInMs: 3000
```

Comment:
I#m a little torn about the timeout since we were told it isn't very accessible, maybe you can make it a bit longer?

Reply:
i think if we do not specify timeInMs it will stay there until user dismisses it. think that is the best way in terms of a11y? @phillimorland otherwise i would increase to 6 sec.?

Reply:
i went with the 6 secs. for now :)